python-version

```diff
@@ -0,0 +1 @@
+3.7.4
```

Comment:
How is this file better than `runtime.txt`? What uses it?

Reply:
This file is used by pipenv. Runtime.txt is used by heroku. If runtime.txt is missing, heroku will generate it from this file. Hence we don't need both of the files together. It may cause inconsistencies.

Reply:
1. Did you mean `pyenv`?

2. Pipenv can also take the python version from `Pipfile`, so if the only other user is `pipenv`, then the argument is lost. 

3. The version is too specific. In development, and CI, any Python 3.7.x will do. Only in production we want to be explicit about using the lastest bugfix release.

Reply:
1. I meant pipenv.

2. You are right we should use Pipefile and runtime.txt to specify the python version and no other file.

3. I think we should keep the version number consistent even to the bug fix level for the following benefits: a) We don't have to blame luck if a minor bug (fixed in a later release) causes any trouble. b) Upgrading the python version frequently will keep reminding us which files to update when doing a python upgrade.

Reply:
3. The problem with specifying a bugfix release is that the entire team (all dev machines), CircleCI & Heroku need to be switched to the exact same bugfix release at the same moment. Which is inconvenient at best, and a huge blocker at worst. 

For example:
3.1. Heroku provides a bugfix release, CircleCI does not. What to do?
3.2. We pin to a bugfix release and one of the developers uses a linux distro that does not yet ship this bugfix release. They lose one day of work in the middle of a busy sprint just setting up their environment and they fail a very important story because of it. 

Both of the above have happened in the past, several times.

Reply:
Okay... Reverting the Pipfile.